src/components/Map.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import '../index.css';
+import * as L from 'leaflet'
+
+
+export default class Map extends React.Component {
```

Comment:
Same comment here for switching to a functional component + TS. The location setting code here could be a good exercise in updating the map due to state changes -- if the user's location changes, can you update the map to follow them? The best approach we've found so far for dealing with that kind of thing is with a heavy use of `useEffect`, like this: https://github.com/azavea/wildfire-hazard-map/blob/develop/src/components/Map.tsx#L68-L150